tests/features/cors/test_cors.py

```diff
@@ -0,0 +1,143 @@
+from tests import TestCase
+from src.masonite.routes import Route
+from src.masonite.middleware import CorsMiddleware
+
+
+class TestCors(TestCase):
+    def setUp(self):
+        super().setUp()
+        # add CorsMiddleware
+        self.application.make("middleware").add([CorsMiddleware])
+        self.cors = self.application.make("cors")
+        self.default_options = {
+            "allowed_origins": ["http://localhost"],
```

Comment:
Good to whichever origin matches an element in this list, thats the host that is returned in the origin header right?

Reply:
I have no idea what I was trying to ask here lol